docs/learn-more/unreachable-commits.md

```diff
@@ -0,0 +1,70 @@
+# Reachable and Unreachable Commits
+
+In Git, every commit will have at least one parent commit except the very first. Additionally, a repository may have any number of branches that begin at any particular commit. Because of this we can create a graph of the history of a commit by following the path from one commit's parent to the another. Given a branch `development`, whose initial commit is `A`, and we add commit `B` and `C`. A resulting graph would be as follows:
+
+```
+development
+C
+|
+B
+|
+A
```

Comment:
Could we use [Mermaid](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/) for this?

```mermaid
    gitGraph
       commit id: "A"
       commit id: "B"
       commit id: "C"
```